crates/shim-kvm/src/snp/ghcb.rs

```diff
@@ -668,8 +666,7 @@ impl GhcbExtHandle {
 
         let vmpck0 = SECRETS.get_vmpck0();
 
-        let key = Key::from_slice(&vmpck0);
-        let cipher = Aes256Gcm::new(key);
+        let cipher = Aes256Gcm::new_from_slice(&vmpck0).unwrap();
```

Comment:
Why an unwrap rather than handling the error?

Reply:
If this does not work, we are in deep trouble, because we can't talk to the firmware with the key, the firmware gave us.
Better panic and bail out. There is no nice way out.